pyspeckit/spectrum/fitters.py

```diff
@@ -273,13 +260,14 @@ def __call__(self, interactive=False, multifit=False, usemoments=True,
             self.start_interactive(clear_all_connections=clear_all_connections,
                                    reset_selection=True,
                                    debug=debug, **kwargs)
-        elif (((multifit or multifit is None) and
-               self.fittype in self.Registry.multifitters)
+        elif (self.fittype in self.Registry.multifitters
               or guesses is not None
               or parinfo is not None):
             if guesses is None and parinfo is None:
-                raise ValueError("You must input guesses when using multifit."
-                                 "  Also, baseline (continuum fit) first!")
+                # raise ValueError("You must input guesses when using multifit."
+                                 # "  Also, baseline (continuum fit) first!")
+                log.warn("""You must input guesses when using multifit. 
+                            Also, baseline (continuum fit) first!""")
```

Comment:
This needs to be reset to an `Exception`.   The fitting will not work without guesses.